src/main/java/frc/robot/subsystems/shooter/ShooterIOTalonFX.java

```diff
@@ -125,6 +125,13 @@ public void setShooterVelocity(double velocity) {
     }
   }
 
+  @Override
+  public void setTargetVelocity(double velocity) {
```

Comment:
Why not use `setVelocity` normally?

Reply:
Is this about the name or the use of setControl?

Reply:
no, in `ShooterIOTalonFX` there is another function called setVelocity, why are we not using that?

Reply:
That’s true but I was slightly worried about caching or something. Ig premature optimization really is the root of all evil. I’ll change that later today